addons/l10n_ua/data/account_tax_group_data.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="tax_group_vat20" model="account.tax.group">
+            <field name="name">ПДВ 20%</field>
+        </record>
+        <record id="tax_group_vat7" model="account.tax.group">
+            <field name="name">ПДВ 7%</field>
+        </record>
+        <record id="tax_group_vat0" model="account.tax.group">
+            <field name="name">ПДВ 0%</field>
+        </record>
+        <record id="tax_group_vat_free" model="account.tax.group">
+            <field name="name">Звільнено від ПДВ</field>
+        </record>
+        <record id="tax_group_not_vat" model="account.tax.group">
+            <field name="name">Не є ПДВ</field>
```

Comment:
should not we just remove then?  If it is not subject to vat then why would you put taxes?

Reply:
same